### DISCUSSION TITLE
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.

### DISCUSSION THREAD

**Discussion by mjdcurtis** [komoot/photon#731] [Q&A] [answered]
Should I be concerned about this error? I'm running openjdk 21-ea 2023-09-19, but I could downgrade the java version to be able to use  sun.reflect.Reflection.getCallerClass. 

Or should I ignore the error?

**Comment by lonvia** [comment, marked as answer]
You should be able to ignore this warning.

**Comment by otbutz** [comment]
@lonvia the warning might have some merit. It looks like the current artifact is missing the `Multi-Release` property in its Manifest. see https://github.com/komoot/photon/pull/732
